cypress/integration/homeFeedFlows/feedNavigation.spec.js

```diff
@@ -26,17 +26,16 @@ describe('Home Feed Navigation', () => {
         'page',
       );
 
-      cy.get('@week').should('have.attr', 'aria-current', '');
-      cy.get('@month').should('have.attr', 'aria-current', '');
-      cy.get('@year').should('have.attr', 'aria-current', '');
-      cy.get('@infinity').should('have.attr', 'aria-current', '');
+      cy.get('@week').should('not.have.attr', 'aria-current');
+      cy.get('@month').should('not.have.attr', 'aria-current');
+      cy.get('@year').should('not.have.attr', 'aria-current');
+      cy.get('@infinity').should('not.have.attr', 'aria-current');
     });
   });
 
   it('should navigate to Week view', () => {
     cy.findByRole('navigation', { name: 'View posts by' }).within(() => {
       cy.findByRole('link', { name: 'Week' }).as('week');
-      cy.get('@week').should('have.attr', 'aria-current', '');
```

Comment:
There's still a couple of tests in this file that need these removals replaced with e.g.

` cy.get('@week').should('not.have.attr', 'aria-current');`

If we can get these added in for everywhere the assertions have been removed, we can get this approved 🚀

Reply:
arf, should be fixed by b8b5c3819dbdd17386668c13b35790707b9bdaaf